app/code/local/Magmodules/Webwinkelconnect/Model/Api.php

```diff
@@ -109,25 +114,44 @@ public function sendInvitation($order)
             CURLOPT_FOLLOWLOCATION => true,
             CURLOPT_FAILONERROR => true,
         ]);
-        $curl->write(Zend_Http_Client::POST, $url, '2', false, $postData);
+        $curl->write(Zend_Http_Client::POST, $url, '2', false, $post_data);
         $response = $curl->read();
 
-        if ($response) {
-            $responseHtml = $response;
-        } else {
-            $responseHtml = sprintf('(%s) %s',$curl->getErrno(), $curl->getError());
+
+        if (!$response) {
+            Mage::getModel('webwinkelconnect/log')->addToLog(
+                'invitation',
+                $order->getStoreId(),
+                '',
+                $helper->__(sprintf('(%s) %s',$curl->getErrno(), $curl->getError())),
```

Comment:
This looks like you're trying to translate a string after applying `sprintf`? That's not going to work, unless the resulting string is coincidentally in the translations CSV. You should probably apply `__` first, and then `sprintf`.